modules/adheseBidAdapter.js

```diff
@@ -18,13 +18,16 @@ export const spec = {
     if (validBidRequests.length === 0) {
       return null;
     }
+    const gdprConsent = bidderRequest.gdprConsent;
+    const refererInfo = bidderRequest.refererInfo;
 
     const account = getAccount(validBidRequests);
     const targets = validBidRequests.map(bid => bid.params.data).reduce(mergeTargets, {});
-    const gdprParams = (bidderRequest.gdprConsent && bidderRequest.gdprConsent.consentString) ? [ 'xt' + bidderRequest.gdprConsent.consentString ] : [];
+    const gdprParams = (gdprConsent && gdprConsent.consentString) ? [ 'xt' + gdprConsent.consentString ] : [];
```

Comment:
In Prebid, we are now requesting that string templates be used in place of string concatenation.

So line 26 should be changed to:
``const gdprParams = (gdprConsent && gdprConsent.consentString) ? [`xt${gdprConsent.consentString}`] : [];``

Line 27 should be changed to:
``const refererParams = (refererInfo && refererInfo.referer) ? [ `xf${base64urlEncode(refererInfo.referer)}`] : [];``

Line 30 should be changed to:
``const params = [...slotsParams, ...targetsParams, ...gdprParams, ...refererParams].map(s => `/${s}`).join('');``